app/routes/api.py

```diff
@@ -1,4 +1,7 @@
 from flask import Blueprint
+from app.models import mega_sena as model
+from flask import jsonify
+from app.schemas import mega_sena as schema
```

Comment:
Mesmo caso do model.

```suggestion
from app.schemas.mega_sena import MegaSenaSchema
```